backends/platform/sdl/kolibrios/kolibrios.mk

```diff
@@ -0,0 +1,35 @@
+bundle = zip-root
+
+all: scummvm.kos $(EXECUTABLE)
+
+scummvm.kos: $(srcdir)/backends/platform/sdl/kolibrios/wrapper-main.c
```

Comment:
You might want to make this a .cpp file since it's already being built by a C++ compiler.

Reply:
Nope. C++ would need statically linking libstdc++ and libsupc++ into the launcher for no reason.